### PR TITLE
✨ We can't be sure device has matching MbedTarget

### PR DESCRIPTION
**Pull request by muchzill4**
### Description

We've established that it's possible that valid Device won't have a matching MbedTarget.

### Test Coverage

- [x]  This change is covered by existing or additional automated tests.
- [ ]  Manual testing has been performed (and evidence provided) as automated testing was not feasible.
- [ ]  Additional tests are not required for this change (e.g. documentation update).

### CODE REVIEW

File: mbed_devices/mbed_tools/cli.py [hunk 1]
```diff
@@ -11,7 +11,8 @@ def _build_output(devices: Iterable[Device]) -> str:
     devices_data = []
     for device in devices:
         mount_points = ", ".join(str(mount_point) for mount_point in device.mount_points)
-        devices_data.append([device.mbed_target.platform_name, device.serial_number, device.serial_port, mount_points])
+        platform_name = device.mbed_target.platform_name if device.mbed_target else "UNKNOWN"
```
Comment (acabarbaye) [inline mbed_devices/mbed_tools/cli.py]: Could we define "UNKNOWN" as a constant which could be used elsewhere in the project?

Comment (acabarbaye) [reply mbed_devices/mbed_tools/cli.py]: should we not do the same for serial?

Comment (muchzill4) [reply mbed_devices/mbed_tools/cli.py]: Good point with serial, I'll change that. I don't think it's worth having such constant yet. Will definitely keep that in mind when looking at presentation layer, is that ok?

Comment (muchzill4) [reply mbed_devices/mbed_tools/cli.py]: Fixed https://github.com/ARMmbed/mbed-devices/pull/24/commits/541a0092c3f24c0ce1974b7b5c7c73652ba303e6.

**Pull request merged**